CHANGELOG.md

```diff
@@ -10,16 +10,17 @@ IMPROVEMENTS:
  * csi: Added periodic garbage collection of plugins and volume claims [[GH-7825](https://github.com/hashicorp/nomad/issues/7825)]
  * csi: Improved performance of volume claim releases by moving work out of scheduler [[GH-7794](https://github.com/hashicorp/nomad/issues/7794)]
  * driver/docker: Added support for custom runtimes [[GH-7932](https://github.com/hashicorp/nomad/pull/7932)]
+ * jobspec: Added support for a per-group policy to stop tasks when a client is disconnected [GH-2185]
```

Comment:
This does qualify as `core` to me as it touches many components beyond simply parsing job spec.

Reply:
I was wavering on that. 😀 Will do.